app/src/main/java/com/kickstarter/libs/utils/AnalyticEventsUtils.kt

```diff
@@ -102,6 +102,7 @@ object AnalyticEventsUtils {
     fun userProperties(user: User, prefix: String = "user_"): Map<String, Any> {
         val properties = HashMap<String, Any>()
         properties["uid"] = user.id()
+        properties["is_admin"] = user.isAdmin ?: false
```

Comment:
if `user.isAdmin` is a boolean, i dont think we need the elvis operator. if not then disregard